utils/abilities_events.cfg

```diff
@@ -3035,7 +3035,7 @@
                         [/filter_location]
                     [/or]
                     [or]
-                        ability_type_active=regenerate
+                                                ability_type_active=regenerate
```

Comment:
I think you messed up indentation here somehow.

Reply:
Yeah, no clue there.  I have to fix that every time I upload that file.  

Unless perhaps you're looking at an earlier commit before I fixed it?